vllm/config.py

```diff
@@ -563,9 +563,13 @@ def __init__(
         if max_num_batched_tokens is not None:
             self.max_num_batched_tokens = max_num_batched_tokens
         else:
-            # If max_model_len is too short, use 2048 as the default value for
-            # higher throughput.
-            self.max_num_batched_tokens = max(max_model_len, 2048)
+            if enable_chunked_prefill:
+                # For chunked prefill, choose the well-tuned batch size.
```

Comment:
This is arbitrary value. We should profile again.

Reply:
actually, there is a no good default -- all depends on application requirements, model, hw, etc. I have an experimental auto-tuner which sets the maximum chunk size that can satisfy the application requirements. It is pretty straight forward, it just takes tbt target and find out the max token budget that can satisfy it.